front/src/common/SpeedLimitByTagSelector/useStoreDataForSpeedLimitByTagSelector.ts

```diff
@@ -39,6 +41,7 @@ export const useStoreDataForSpeedLimitByTagSelector = () => {
     }
   }, [error]);
 
+  const speedLimitsByTags = compact(uniq(concat(speedLimitTags, speedLimitsTagsByInfraId)));
```

Comment:
Maybe you could inject directly here the `DEFAULT_SPEED_LIMIT_TAG` in the list instead of doing that multiple times in other components. What do you think ?

It seems the only component where you don't need the default value is in the infra editor so over there you could just `shift` the array to remove the first element.